packages/grafana-ui/src/components/Forms/FieldSet.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, HTMLProps } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory, useTheme } from '../../themes';
+import { Legend } from './Legend';
+
+export interface Props extends HTMLProps<HTMLFieldSetElement> {
+  children: React.ReactNode[] | React.ReactNode;
+  /** Text for the fieldset's legend */
+  label?: string;
+}
+
+export const FieldSet: FC<Props> = ({ label, children, className, ...rest }) => {
```

Comment:
This is more of a philosphivsl consideration than implementation one. Should it be called `FieldSet` or `Fieldset`? Intuition tells me `Fieldset`: HTML element is called `fieldset` and following react components naming convention I would call it `Fieldset`. But, there is HTML spec which has this: https://www.w3.org/TR/html52/sec-forms.html#htmlfieldsetelement :D and I'm confused 😕

Reply:
Interesting point. Not sure why html `fieldset` is all lowercased, since it's technically two words, but seems like it's some kind of convention (same goes for `dataset`, etc). So the ultimate decision should be based on whether we consider fieldset to be one word or two words, for one word it's then `Fieldset`, two words would be `FieldSet`. At least my IDE's spellchecker considers it to be a composite of two words :D